dev/release/post-website.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+
+SOURCE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ARROW_DIR="${SOURCE_DIR}/../../"
+: ${ARROW_SITE_DIR:="${ARROW_DIR}/../arrow-site"}
+
+if [ "$#" -ne 2 ]; then
+  echo "Usage: $0 <previous-version> <version>"
+  exit 1
+fi
+
+previous_version=$1
+version=$2
+
+branch_name=release-note-arrow-go-${version}
+release_dir="${ARROW_SITE_DIR}/_posts"
+announce_file="${release_dir}/$(date +%Y-%m-%d)-arrow-go-${version}.md"
+
+pushd "${ARROW_SITE_DIR}"
+DEFAULT_BRANCH="$(git rev-parse --abbrev-ref origin/HEAD | sed s@origin/@@)"
+git fetch --all --prune --tags --force -j$(nproc)
+git checkout ${DEFAULT_BRANCH}
+git branch -D ${branch_name} || :
+git checkout -b ${branch_name}
+popd
+
+pushd "${ARROW_DIR}"
+
+previous_major_version="$(echo v${previous_version} | grep -o '^[0-9]*')"
+major_version="$(echo v${version} | grep -o '^[0-9]*')"
+if [ ${previous_major_version} -eq ${major_version} ]; then
+  release_type=patch
+else
+  release_type=major
+fi
+
+export TZ=UTC
+release_date=$(LC_TIME=C date "+%-d %B %Y")
+release_date_iso8601=$(LC_TIME=C date "+%Y-%m-%d")
+previous_tag_date=$(git log -n 1 --pretty=%aI v${previous_version})
+rough_previous_release_date=$(date --date "${previous_tag_date}" +%s)
+rough_release_date=$(date +%s)
+rough_n_development_months=$(((\
+  ${rough_release_date} - ${rough_previous_release_date}) / (60 * 60 * 24 * 30)))
+
+git_tag=v${version}
+git_range=v${previous_version}..v${version}
+
+contributors_command_line="git shortlog -sn ${git_range}"
+contributors=$(${contributors_command_line} | grep -v dependabot)
+
+n_commits=$(git log --pretty=oneline ${git_range} | grep -i -v "chore: Bump" | wc -l)
+n_contributors=$(${contributors_command_line} | grep -v dependabot | wc -l)
+
+git_tag_hash=$(git log -n 1 --pretty=%H ${git_tag})
+git_changelog="$(gh release view --json body --jq .body | grep -v '@dependabot')"
```

Comment:
Could you replace `##` section markup to `###` because we use `## Changelog` section?

```suggestion
git_changelog="$(gh release view --json body --jq .body | grep -v '@dependabot' | sed -e 's/^#/##/g')"
```